src/middlewared/middlewared/plugins/failover_/disabled_reasons.py

```diff
@@ -14,6 +14,7 @@ class DisabledReasonsEnum(str, Enum):
     NO_CRITICAL_INTERFACES = 'No network interfaces are marked critical for failover.'
     MISMATCH_DISKS = 'The quantity of disks do not match between the nodes.'
     MISMATCH_VERSIONS = 'TrueNAS software versions do not match between storage controllers.'
+    MISMATCH_NICS = 'NIC hardware does not match between nodes.'
```

Comment:
Since this is public facing verbiage, we need to get with our documentation team on some guidance of what to use. For example, "NIC" isn't very proper since acronyms are ambiguous at best. @DjP-iX or @Mrt134 any suggestions on what we should use here?

I had the idea of using similar verbiage for other messages. i.e. `Network interfaces do not match between storage controllers.`